### DISCUSSION TITLE
Translation to Chinese

### DISCUSSION THREAD

**Discussion by hhk-png** [browserengineering/book#1139] [Ideas]
Hi! Thanks for the awesome content!

Since we don't have as much tech content in chinese compared to english, would you guys allow for me to translate it to Chinese? And what rules should I follow if I want to translate this book?

**Comment by chrishtr** [comment]
> Thanks for the awesome content!

Thank you!

> Since we don't have as much tech content in chinese compared to english, would you guys allow for me to translate it to Chinese? And what rules should I follow if I want to translate this book?

We have a publisher and it'd have to be a contract to buy rights for Chinese translations unfortunately. Feel free to email us if you're interested in that.

**Reply by hhk-png** [reply]
Thanks for your reply! But there is no enough resources in my hand to buy the rights. Hope one day I could this.
